### PR TITLE
Jakarta WebSocket 2.1 Plan Review

### PR DESCRIPTION
**Pull request by markt-asf**
## Plan Review PR template
When creating a specification project plan review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version.
- [x] An index page wombat/x.y/_index.md following [template](https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md)
- [x] Title should be on the form "Jakarta Wombat X.Y (under development)"
- [x] Includes a plan for the release. Usually, a couple of statements is enough. Otherwise, a link to a document describing the release.
- [x] Includes a detailed description of any backward incompatibility (Mark with **N/A** and check if none)
- [x] Declares optional features if there are any (Mark with **N/A** and check if none)
- [x] Includes minimum Java SE version
- [x] Describe any updates or required changes including splitting out the TCK (Mark with **N/A** and check if not planned)
- [x] Link to updated release record

### CODE REVIEW

File: websocket/2.1/_index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: "Jakarta WebSocket 2.1 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta WebSocket defines a API for Server and Client Endpoints
+for the WebSocket protocol (RFC6455).
+
+* [Jakarta WebSocket 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.websocket/releases/2.1)
```
Comment (kazumura) [inline websocket/2.1/_index.md:9]: @markt-asf   
Could you delete the lines from the Release Record link to the maven artifact link in '_index.md' ?  
These lines would be added for Progress- and/or Release Reviews.

**Review by kazumura — CHANGES_REQUESTED** [review]
@markt-asf   Thanks.
It seems to be almost OK. But there is one thing to be changed.

File: websocket/2.1/_index.md (hunk 1 above)
Comment (markt-asf) [reply websocket/2.1/_index.md:9]: Done.

**Review by kazumura — APPROVED** [review]

**Comment by kazumura** [comment]
The ballot has been started.  
https://www.eclipse.org/lists/jakarta.ee-spec/msg01613.html

File: websocket/2.1/_index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```
Comment (kwsutter) [inline websocket/2.1/_index.md:10]: I would drop this line.  Or, if you want to make an educated guess on where the Jakarta EE 10 Release Plan will be...  ;-)

File: websocket/2.1/_index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+## Plan Review
```
Comment (kwsutter) [inline websocket/2.1/_index.md:60]: I would put the Plan Review ballot results before the Release Review ballot results.  But, maybe that's just me...

File: websocket/2.1/_index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
```
Comment (kwsutter) [inline websocket/2.1/_index.md:59]: You are missing the reference to the Spec mailing list url where the vote took place, like what WebSocket 2.0 provided...  `This ballot was conducted on the public e-mail list [jakarta.ee-spec@eclipse.org](). This ballot thread begins [here]().`

File: websocket/2.1/_index.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+|-----------------------|---------|-----|----------|
+|Fujitsu                |         |     |          |
+|IBM                    |         |     |          |
+|Oracle                 |         |     |          |
+|Payara                 |         |     |          |
+|Red Hat                |         |     |          |
+|Tomitribe              |         |     |          |
+|EE4J PMC               |         |     |          |
+|Participant Members    |         |     |          |
+|Committer Members      |         |     |          |
```
Comment (kwsutter) [inline websocket/2.1/_index.md:74]: Same comment as above for the Release Review reference for the mailing list ballot thread.

**Review by kwsutter — COMMENTED** [review]
@markt-asf I'll just leave these as comments and abstain from approving.  I trust you'll make these updates before the PR gets merged.  Thanks!

**Comment by kazumura** [comment]
Ballot result:  
https://www.eclipse.org/lists/jakarta.ee-spec/msg01704.html

**Pull request merged**